generators/lib/DnnGeneratorBase.js

```diff
@@ -77,7 +77,7 @@ module.exports = class DnnGeneratorBase extends Generator {
   }
 
   _getNamespace() {
-    let namespace = this.props.company;
+    let namespace = this.props.namespace;
```

Comment:
@VAlmea This is a pretty global change, but I don't see any other related updates. Do the other templates still work with this PR?

Reply:
After this update I tried generating a WebForm template with no problems, I'll check the rest.

Reply:
@VAlmea What was the result of your check?

Reply:
@WillStrohl I revert these changes